containers/signing-service/packages.txt

```diff
@@ -0,0 +1,7 @@
+gcc
```

Comment:
I'd call the file `build-dependencies.txt` or something like this to make it clear that the file contains packages that will be uninstalled after the build.

We don't want someone to add his package here and discover later that the container does not work becuause an important dependency has been removed after the build.

Reply:
Done.